src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/step_time_field/step_time_field.html

```diff
@@ -2,173 +2,214 @@
+    </div>
+    <div class="euiFormControlLayout">
+      <input
+        class="euiFieldText euiFieldText--large"
+        id="indexPatternCreationId"
+        data-test-subj="createIndexPatternIdInput"
+        ng-model="stepTimeField.indexPatternId"
+        ng-model-options="{ updateOn: 'default blur', debounce: {'default': 500, 'blur': 0} }"
+        validate-index-pattern
+        validate-index-pattern-allow-wildcard
+        name="id"
+        type="text"
       >
+    </div>
+    <div class="euiFormHelpText euiFormRow__text">
+      <span class="euiTextColor euiTextColor--subdued">
         Kibana will provide a unique identifier for each index pattern.
         If you do not want to use this unique ID, enter a custom one.
-      </p>
+      </span>
     </div>
+  </div>
 
-    <!-- Actions -->
-    <div class="kuiBar kuiVerticalRhythm">
-      <div class="kuiBarSection">
-        <div class="kuiButtonGroup">
-          <button
-            class="kuiButton kuiButton--secondary kuiButton--iconText"
-            ng-click="stepTimeField.goToPreviousStep()"
-          >
-            <span class="kuiButton__inner">
-              <span aria-hidden="true" class="kuiButton__icon kuiIcon fa-chevron-left"></span>
-              <span>Back</span>
-            </span>
-          </button>
+  <div class="euiSpacer euiSpacer--m"></div>
 
-          <button
-            data-test-subj="createIndexPatternCreateButton"
-            ng-disabled="!stepTimeField.canCreateIndexPattern()"
-            class="kuiButton kuiButton--primary"
-            type="submit"
-          >
-            Create index pattern
-          </button>
-        </div>
-      </div>
+  <!-- Actions -->
+  <div class="euiFlexGroup euiFlexGroup--gutterSmall euiFlexGroup--justifyContentFlexEnd">
+    <div class="euiFlexItem euiFlexItem--flexGrowZero">
+      <button
+        class="euiButtonEmpty euiButtonEmpty--primary kuiButton--iconText"
+        ng-click="stepTimeField.goToPreviousStep()"
+      >
+        <span class="euiButtonEmpty__content">
+          <svg aria-hidden="true" class="euiIcon euiIcon--medium" xmlns="http://www.w3.org/2000/svg"
+            xlink="http://www.w3.org/1999/xlink" width="16" height="16" viewBox="0 0 16 16">
+            <defs>
+              <path id="arrow_left-a" d="M13.069 5.157L8.384 9.768a.546.546 0 0 1-.768 0L2.93 5.158a.552.552 0 0 0-.771 0 .53.53 0 0 0 0 .759l4.684 4.61c.641.631 1.672.63 2.312 0l4.684-4.61a.53.53 0 0 0 0-.76.552.552 0 0 0-.771 0z"
+              />
+            </defs>
+            <use fill-rule="nonzero" transform="rotate(90 8 8)" href="#arrow_left-a"
+            />
+          </svg>
+          <span>Back</span>
+        </span>
+      </button>
+    </div>
+    <div class="euiFlexItem euiFlexItem--flexGrowZero">
+      <button
+        data-test-subj="createIndexPatternCreateButton"
+        ng-disabled="!stepTimeField.canCreateIndexPattern()"
+        class="euiButton euiButton--primary"
```

Comment:
Can we add `euiButton--fill` here to make it clearer that this is the button that completes this process?